<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreferences</class>
 <widget class="QDialog" name="DialogPreferences">
  <property name="windowTitle" stdset="0">
   <string>Dialog</string>
  </property>
  <property name="geometry" stdset="0">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>254</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelFontSize">
         <property name="text" stdset="0">
          <string>Font Size: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxApplicationFontSize">
         <property name="value" stdset="0">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxEventStars">
         <property name="text" stdset="0">
          <string>Event Stars</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxDarkStyle">
         <property name="text" stdset="0">
          <string>Dark Style</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButtonTodayColour">
         <property name="text" stdset="0">
          <string>Today Colour</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEventColour">
         <property name="text" stdset="0">
          <string>Event Colour</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPriorityColour">
         <property name="text" stdset="0">
          <string>Priority Colour</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPublicColour">
         <property name="text" stdset="0">
          <string>Public Holiday Colour</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBoxPublicHolidays">
         <property name="text" stdset="0">
          <string>UK Public Holidays</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxAmPmFormat">
         <property name="text" stdset="0">
          <string>AM/PM Format</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxAudioAlarm">
         <property name="text" stdset="0">
          <string>Talk Alarm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons" stdset="0">
          <set>QDialogButtonBox::NoButton</set>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxTalkStartup">
         <property name="text" stdset="0">
          <string>Talk At Startup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxUpcoming">
         <property name="text" stdset="0">
          <string>Talk Upcoming</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text" stdset="0">
          <string>Upcoming Days:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxUpcomingDays"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="checkBoxFactoryReset">
         <property name="text" stdset="0">
          <string>Factory Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_12">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_13">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation" stdset="0">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonOk">
           <property name="text" stdset="0">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancel">
           <property name="text" stdset="0">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
